packages/minifier-css/minifier.js

```diff
@@ -63,25 +63,35 @@ const CssTools = {
    *
    * @param {string} cssText CSS string to minify.
    * @return {String[]} Array containing the minified CSS.
+   * @deprecated on 2.8
    */
   minifyCss(cssText) {
-    const f = new Future;
-    postcss([
-      cssnano({ safe: true }),
-    ]).process(cssText, {
-      from: void 0,
-    }).then(result => {
-      f.return(result.css);
-    }).catch(error => {
-      f.throw(error);
-    });
-    const minifiedCss = f.wait();
-
+    const f = new Future();
+    CssTools.minifyCssAsync(cssText)
+      .then((res) => f.return(res))
+      .catch((error) => f.throw(error));
     // Since this function has always returned an array, we'll wrap the
     // minified css string in an array before returning, even though we're
     // only ever returning one minified css string in that array (maintaining
     // backwards compatibility).
-    return [minifiedCss];
+    return f.wait();
```

Comment:
@zodern Can we use `Promise.await` in here (i.e., build plugins)? It'd make the code much cleaner.

Reply:
Yes, we can.

Reply:
I did it.

Thank you for the suggestion!